godotproject/Menus/InGameMenu/MusicVolumeSlider.gd

```diff
@@ -1,5 +1,9 @@
 extends HSlider
 
+func set_volume(volume):
+	value = volume
+	Global.music_volume = value
```

Comment:
do we want to modify the music volume here or the master? I could see it being a pain point, turning the volume all the way off in the title screen and then going in and still hearing sound effects

Reply:
It affects both sound effect and music buses, they will both be set to the title screen volume when you start playing, and then you can adjust them independently. When returning to the title screen, it'll always set them back to a low level (3/10)